src/config.ts

```diff
@@ -359,6 +359,11 @@ export const ARNS_ON_DEMAND_CIRCUIT_BREAKER_RESET_TIMEOUT_MS =
     `${5 * 60 * 1000}`, // 5 minutes
   );
 
+export const ARNS_NAMES_CACHE_TTL_SECONDS = +env.varOrDefault(
+  'ARNS_NAMES_CACHE_TTL_SECONDS',
+  `${60 * 60}`, // 1 hour
```

Comment:
Let's default this to 5 minutes. We don't want it to be too long since that'll impact how long it takes new names to be available on gateways.

Reply:
@dtfiedler and I discussed about this and we think 1 hour is a nice TTL as the goal is to reduce CU load with these requests and a name cannot be prescribed until the epoch after it has been purchased

Reply:
@karlprieb - i pull back my original suggestion of an hour. as name purchases ramp up this could cause some frustration of names not resolving quickly. 5 mins sounds good